python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -1080,6 +1149,15 @@ router bgp {{ router_bgp.as }}
 {%             elif neighbor.activate is arista.avd.defined(false) %}
       no neighbor {{ neighbor.ip_address }} activate
 {%             endif %}
+{%             if neighbor.additional_paths.send is arista.avd.defined %}
```

Comment:
same as ipv4 AFI comment, missed the receive  and install/install_ecmp_primary logic

Reply:
Fixed.